<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Locar_Cadastro_Veiculo</class>
 <widget class="QMainWindow" name="Locar_Cadastro_Veiculo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Locar</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(85, 255, 255);
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="subtitulo_console">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>100</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>700</width>
      <height>100</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>700</width>
      <height>100</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="lineWidth">
     <number>8</number>
    </property>
    <property name="midLineWidth">
     <number>5</number>
    </property>
    <property name="text">
     <string>Console de Gerenciamento Locar</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Forma de Pagamento</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label6">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>340</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>Classe do Veiculo</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>310</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>CPF do Cliente</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>280</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>Quantidade de Diarias</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>410</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Serviços Adicionais</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>220</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>Data Devolução (ANO/MES/DIA)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>300</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Data Retirada (ANO/MES/DIA)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>260</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Exemplo: Ano/Mes/Dia - 2022/02/20</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>260</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Exemplo: Ano/Mes/Dia - 2022/02/20</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>280</y>
      <width>260</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>310</y>
      <width>260</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>Somente os numeros</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>560</y>
      <width>100</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>50</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label7_2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>630</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>20</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);</string>
    </property>
    <property name="text">
     <string>Valor Total:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label7_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>630</y>
      <width>90</width>
      <height>40</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>90</width>
      <height>40</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>90</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 255, 127);</string>
    </property>
    <property name="text">
     <string>Cadastro de Reserva</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>650</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
border-color: rgb(255, 0, 255);</string>
    </property>
    <property name="text">
     <string>SAIR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 255);
background-color: rgb(0, 170, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>TELA INICIAL</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>260</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
    <item>
     <property name="text">
      <string>Cartão de Crédito</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Cartão de Débito</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Dinheiro</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>340</y>
      <width>260</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>20</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
    <item>
     <property name="text">
      <string>Sedan</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hatch</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SUV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Utilitário</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Caminhonete</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sportivo</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>Seguro Veicular Acidente</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>500</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>Seguro Veicular Completo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>470</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Seguro Veiculo Roubo e Furto</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>440</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Cadeira de Bebê</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>500</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>Condutor Extra</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>470</y>
      <width>171</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="text">
     <string>Bebê Conforto</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>470</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
    <item>
     <property name="text">
      <string>Não</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sim</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>440</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
    <item>
     <property name="text">
      <string>Não</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sim</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>500</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
    <item>
     <property name="text">
      <string>Não</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sim</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_6">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>440</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
    <item>
     <property name="text">
      <string>Não</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sim</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_7">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>470</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
    <item>
     <property name="text">
      <string>Não</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sim</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_8">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>500</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
    <item>
     <property name="text">
      <string>Não</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sim</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
